### PR TITLE
add angular-bootstrap-grid-tree@0.4.0 w/npm auto-update #9478

### LINKED ISSUE #9478: [Request] Add tree-grid-directive
**Library name:** tree-grid-directive
**Git repository url:** https://github.com/khan4019/tree-grid-directive
**npm package url(optional):** $ https://www.npmjs.com/package/angular-bootstrap-grid-tree
**License(s):** MIT license
**Official homepage:** http://khan4019.github.io/tree-grid-directive/test/treeGrid.html
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by emmy41124**
close #9478, cc @khan4019

Pull request for issue: #9478 

@nolsherry please help me review this PR, thank you
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/khan4019/tree-grid-directive
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/angular-bootstrap-grid-tree
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 278
   - Count of watchers: 39
   - Count of forks: 151
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 35
   - Downloads in the last month: 288

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/angular-bootstrap-grid-tree/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "author": {
+    "name": "Jhankar Mahbub",
+    "url": "https://github.com/khan4019"
+  },
+  "description": "AngularJS directive for a tree grid, using Bootstrap",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/khan4019/tree-grid-directive.git"
+  },
+  "dependencies": {},
```
Comment (nolsherry) [inline ajax/libs/angular-bootstrap-grid-tree/package.json]: Please remove `dependencies` field, thanks!

**Comment by PeterDaveHello** [comment]
Any updates here?

**Comment by nolsherry** [comment]
LGTM, @cdnjs/intern2 please help me review this PR, thanks!

File: ajax/libs/angular-bootstrap-grid-tree/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "grid",
+    "widget"
+  ],
+  "license": "MIT",
+  "npmName": "angular-bootstrap-grid-tree",
+  "npmFileMap": [
+    {
+      "basePath": "src",
+      "files": [
+        "tree*"
```
Comment (kennynaoh) [inline ajax/libs/angular-bootstrap-grid-tree/package.json]: I think `+(tree-grid-directive*|treeGrid*)` is better.

**Pull request closed without merging**